app/scrapers/google.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import print_function
+import os, json, sys
+import requests
+from bs4 import BeautifulSoup
+
+class Google:
+    'Scrapper class for Google'
```

Comment:
please do this with `"""`, and you might as well add a doc string for each class